### PR TITLE
[11.0][MIG] privacy_partner_report

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by mreficent** [comment]
WIP

**Comment by HviorForgeFlow** [comment]
Images are displayed in str convert from byte format. I'm not familiar with image manipulation in Odoo. How could it be handle this??

**Comment by HviorForgeFlow** [comment]
export XLSX on runbot is failing due to:
image

File: privacy_partner_report/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import json
+from odoo import http
+
+from odoo.addons.web.controllers.main import serialize_exception
+from odoo.addons.web.controllers.main import Reports
+
+
+class ReportsExtended(Reports):
+    # HACK of https://github.com/odoo/odoo/pull/24964
```
Comment (AaronHForgeFlow) [inline privacy_partner_report/controllers/main.py]: There are changes in Odoo out of that PR. The report seems working without this hack now.

**Comment by AaronHForgeFlow** [comment]
Is it possible/needed to change this parameter?
http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers

No needed, the problem is in the report.

**Comment by aheficent** [comment]
It seems point_of_sale module had the same issue:
https://github.com/odoo/odoo/commit/f4200f376bbd90e919073100170853ada4ad71fc

**Comment by AaronHForgeFlow** [comment]
It seems passing data to the report using the report_action function is not possible right now. It was causing  the controller to do a huge POST.

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Please, squash

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: fine from my side

**Comment by AaronHForgeFlow** [comment]
@mreficent squashed and rebased

**Review by angelmoya — APPROVED** [review]
:+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**